Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -2464,14 +2463,13 @@ void MolDraw2D::extractBondNotes(const ROMol &mol) {
     std::string note;
     if (bond->getPropIfPresent(common_properties::bondNote, note)) {
       if (!note.empty()) {
-        auto note_rect = calcAnnotationPosition(mol, bond, note);
-        if (note_rect.width_ < 0.0) {
+        AnnotationType annot;
+        annot.text_ = note;
+        calcAnnotationPosition(mol, bond, annot);
+        if (annot.rect_.width_ < 0.0) {
           cerr << "Couldn't find good place for note " << note << " for bond "
                << bond->getIdx() << endl;
```

Comment:
Again, not part of this PR, but:
```suggestion
          BOOST_LOG(rdWarningLog) << "Couldn't find good place for note " << note << " for bond "
               << bond->getIdx() << std::endl;
```